### ISSUE TITLE
Workspace not always saved

### ISSUE THREAD

**Issue by EwenG** [joshuafcole/recall#10]
Hi, I noticed that my workspace is not saved when lighttable is closed through the "File -> Close window" menu entry.

**Comment by joshuafcole** [comment]
Interesting. Can you give some platform information? I'll try to replicate
this behavior tonight and see if I can find a better way to detect window
closing through LT's triggers now that it is open source.

On Mon, Jan 13, 2014 at 1:32 PM, EwenG notifications@github.com wrote:

> Hi, I noticed that my workspace is not saved when lighttable is closed
> through the "File -> Close window" menu entry.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/joshuafcole/recall/issues/10
> .

## 

Screw the environment. Please print this email immediately. And then burn
it.

**Comment by EwenG** [comment]
I tried on both windows8 and linux. Same issue.

**Comment by joshuafcole** [comment]
Going to follow up on this tonight and tomorrow. I've been integrating support for commands and behaviors into the underlying JS API used by recall. Hopefully by making recall use actual behaviors and attaching those to the appropriate tags we can ensure that LT triggers it at the appropriate times instead of relying on apparently flimsy DOM event signals.

**Comment by joshuafcole** [comment]
Sorry for not following up earlier, things have been very busy at my startup. I'm unable to reproduce this behavior. Are you still experiencing the issue? If so can you give some more detailed reproduction steps on things that might be related?

**Comment by paxidently** [comment]
+1. I use Linux (Kubuntu 14.04 64bit) and LT 0.7.2 (bin 0.8.4). When I closing LT using 'Close window' menu, workspace does not restoring at start. When I closing it using window title button, all works fine.

**Comment by Xananax** [comment]
I use Arch Linux.

I'm not sure if my problem is the same, related, completely different, or if I'm just an idiot; but my workspace and/or files are just never recalled. Regardless of the method used to close Light Table, I need to "open recent workspace" and reopen each file every time.

Also, not sure if this plugin is supposed to help with that, but Light Table doesn't remember my window size settings, it seems to open at whatever it's default size is, regardless of if I resized it or maximized or what. It's an extremely minor inconvenience, but I thought I'd mention it.

Am I just being obtuse? Is there something to do beyond just installing the plugin? I tried looking for options or settings, and couldn't find anything, whether in the light Table UI, or in the plugin description.
